docs/VisualizingECL/VizMods/VizIntro.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<chapter id="VizIntro">
+  <title><emphasis>Visualization Introduction</emphasis></title>
+
+  <para>The Visualization bundle is an open-source add-on to the HPCC platform
+  to allow you to create visualizations from the results of queries written in
+  ECL.</para>
+
+  <para>Visualizations are an important means of conveying information from
+  massive data. A good visual representation can help a human produce
+  actionable analysis. A visually comprehensive representation of the
+  information can help make the obscure more obvious.</para>
+
+  <para>Pie Charts, Line graphs, Maps, and other visual graphs help us
+  understand the answers found in data queries. Crunching big data is only
+  part of a solution; we must be able to make sense of the data, too. Data
+  visualizations simplify the complex.</para>
+
+  <para>The Visualizer bundle extends the HPCC platform's functionality by
+  allowing you to plot your data onto charts, graphs, and maps to add a visual
+  representation that can be easily understood.</para>
+
+  <para>In addition, the underlying visualization framework supports advanced
+  features to allow you to combine graphs to make interactive
+  dashboards.</para>
+
+  <sect1 id="VizInstall">
+    <title>Installation</title>
+
+    <para>The Visualization bundle is an open-source add-on to the HPCC
+    platform to allow you to create visualizations from the results from
+    queries written in ECL.</para>
+
+    <para>To install, use the ecl command line interface.</para>
+
+    <para></para>
+
+    <para><programlisting>ecl bundle install https://github.com/hpcc-systems/Visualizer.git</programlisting></para>
+
+    <para>To use the "ecl bundle install &lt;git url&gt;" command, git must be
+    installed on your machine and accessible to the user (in the path).</para>
+
+    <para></para>
+  </sect1>
+
+  <sect1 id="VizUsing">
+    <title>Using the Visualization library</title>
+
+    <para>Once installed, you merely IMPORT the library, then call any method
+    that is appropriate for your data shape.</para>
+
```

Comment:
Creating a basic visualization typically requires the following steps:
1.  Creation of a suitable dataset
2.  Output the dataset with a suitable name, so that visualization can locate the data.
3.  Create (and output) a the visualization, referencing the named output from step 2 

To change the appearance of the visualization "dermatology properties" can provided while creating the visualization or can be applied at runtime via the live dermatology property editor.